readme.md

```diff
@@ -40,124 +40,129 @@ $ npm -v
 
 * **Your Answer:**
 
----
+_forking is just a request for cloning_
```

Comment:
It's a bit more than that. I would review the lesson video from yesterday. I explain this more in depth at the beginning of class.